### PR TITLE
Wishlist promotion task

### PR DESCRIPTION
**Pull request by scottb**
Adds a rake task to promote approved wishlist tasks to the main board.

Removes the `before_save` hook to clear the approved status; this is handled by the rake task.

Upgrades the sqlite3 gem to make `bundle-audit` happy. Disables `audit` in lefthook because it's too aggressive.

### CODE REVIEW

**Review by zoeblairfriedman — CHANGES_REQUESTED** [review]
With this change, if you move a task through the UI from one board to another, the approved status will not be cleared—this is misleading as the "approved" value means different things on our two different boards. "Approved by Ron" to be tackled on the wishlist vs "Reviewed" on the default board. Can we either add something back in that resets the value on a board change or remove the ability to change boards from the UI?

**Comment by scottb** [comment]
I've modified the update path in the tasks controller to clear the approved flag if the board changes, and the flag isn't explicitly changed on the form. There's a new spec in `spec/requests/task_spec.rb` that verifies this functionality.

File: spec/requests/tasks_spec.rb [hunk 1]
```diff
@@ -101,6 +101,18 @@
       expect { subject }.to(change { task.reload.title })
     end
 
+    context "when the board is changed" do
+      subject { put task_path(task), params: }
+
+      let(:wishlist) { create(:wishlist) }
+      let(:task) { create(:task, approved: true, board: wishlist) }
+      let(:params) { { task: { title: "new title" } } }
```
Comment (zoeblairfriedman) [inline spec/requests/tasks_spec.rb:109]: Are these params necessary? They make the test a little confusing.

**Review by zoeblairfriedman — APPROVED** [review]
Approved with one suggestion

**Review by m4thayus — APPROVED** [review]

**Pull request merged**